runtime/gc_base/RootScanner.cpp

```diff
@@ -241,13 +239,13 @@ MM_RootScanner::doStringTableSlot(J9Object **slotPtr, GC_StringTableIterator *st
 	doSlot(slotPtr);
 }
 
-#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+#if defined(J9VM_GC_SPARSE_HEAP_ALLOCATION)
 void
-MM_RootScanner::doDoubleMappedObjectSlot(J9Object *objectPtr, struct J9PortVmemIdentifier *identifier)
+MM_RootScanner::doObjectInVirtualLargeObjectHeap(J9Object *objectPtr, bool *sparseHeapAllocation)
```

Comment:
we it be cleaner if this method return a bool (object died?) and the caller sets sparseHeapAllocation back to false